ui/ui-components/hooks/useApi.ts

```diff
@@ -10,14 +10,14 @@ export function UseApi(
   errorHandler?: ErrorHandler,
   uploadHandler?: UploadHandler
 ) {
-  async function execApi(
+  async function execApi<T extends {} = any>(
```

Comment:
I'm looking forward to the day that T is required - that will ensure our API usage is typed!